src/hextools/germ/ophyd.py

```diff
@@ -248,10 +265,15 @@ def trigger(self):
         current_frame = self.frame_num.get()
         self.count.put(AcqStatuses.ACQUIRING.value)
 
-        datum_document = self._datum_factory(datum_kwargs={"frame": current_frame})
-        self._asset_docs_cache.append(("datum", datum_document))
+        stream_datum_document = self._stream_datum_factory(
+            StreamRange(start=current_frame - 1, stop=current_frame),
+        )
 
-        self.image.put(datum_document["datum_id"])
+        print(f"stream_datum_document:\n{pformat(stream_datum_document)}")
+
+        self._asset_docs_cache.append(("stream_datum", stream_datum_document))
+
+        # self.image.put(stream_datum_document["datum_id"])
```

Comment:
I think we need to fix it - @danielballan, @genematx, do we need it now with the Tiled backend?

Reply:
Discussed it with @danielballan in person, it's not needed anymore.